bundles/org.openhab.binding.groupepsa/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,559 @@
+	<channel-type id="autonomy">
+		<item-type>Number:Length</item-type>
+		<label>Fuel Autonomy</label>
+		<description>Remaining distance</description>
+		<category>Pressure</category>
+		<state pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="consumption">
+		<item-type>Number:VolumetricFlowRate</item-type>
+		<label>Fuel Consumption</label>
+		<description>Fuel consumption</description>
+		<category>Pressure</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="residual">
+		<item-type>Number:Energy</item-type>
+		<label>Remaining Charge</label>
+		<description>Remaining battery charge</description>
+		<category>BatteryLevel</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="capacity">
+		<item-type>Number:Energy</item-type>
+		<label>Battery Capacity</label>
+		<description>Battery capacity</description>
+		<category>BatteryLevel</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="healthCapacity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Battery Health - Capacity</label>
+		<description>Health of the battery capacity</description>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="healthResistance">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Battery Health - Resistance</label>
+		<description>Health of the battery resistance</description>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="chargingStatus">
+		<item-type>String</item-type>
+		<label>Charging Status</label>
+		<description>Battery charging status</description>
+		<state readOnly="true">
+			<options>
+				<option value="Disconnected">Disconnected</option>
+				<option value="InProgess">In Progress</option>
+				<option value="Failure">Failure</option>
+				<option value="Stopped">Stopped</option>
+				<option value="Finished">Finished</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="chargingMode">
+		<item-type>String</item-type>
+		<label>Charging mode</label>
+		<description>Battery charging mode</description>
+		<state readOnly="true">
+			<options>
+				<option value="No">No</option>
+				<option value="Slow">Slow</option>
+				<option value="Quick">Fast</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="chargingPlugged">
+		<item-type>Contact</item-type>
+		<label>Plugged In</label>
+		<description>Vehicle plugged in to charger</description>
+	</channel-type>
+
+	<channel-type id="chargingRate">
+		<item-type>Number:Speed</item-type>
+		<label>Charging Rate</label>
+		<description>Battery Charging Rate</description>
+		<category>BatteryLevel</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="chargingRemainingTime">
+		<item-type>Number:Time</item-type>
+		<label>Charging Time Remaining</label>
+		<description>Time remaining till charged</description>
+		<category>Time</category>
+		<state pattern="%tT"/>
```

Comment:
Are you sure this formats nicely?

Reply:
It gives me full date and time. For my Dutch system: "zondag, 24 apr. 2022 18:00".